packages/fela-plugin-extend/src/index.js

```diff
@@ -22,7 +22,10 @@ function removeUndefined(style: Object): Object {
 
 function extendStyle(style: Object, extension: Object): void {
   // extend conditional style objects
-  if (extension.hasOwnProperty('condition')) {
+  if (
+    extension &&
+    Object.prototype.hasOwnProperty.call(extension, 'condition')
```

Comment:
What's the benefit of calling it this way instead of just `extension.hasOwnProperty('condition')`?

Reply:
It's not a big deal, but if someone uses `Object.create(null)` to create a lightweight map without the entire object prototype, `.hasOwnProperty` would not exist and the code would throw an `hasOwnProperty is not a function` TypeError.

So, if somene did:
```js
<FelaComponent 
  style={Object.assign(Object.create(null), { extend: [ /* ... */ ], })}
>
```

that would error out.

I wouldn't necessarily bother with this kind of thing in app code, but I feel library code should be as robust as possible and I don't really see any down side to it.

Reply:
Let me know if you'd like me to change it back or release it like this

Reply:
Oh sure not, that makes sense - didn't know it =)